### DISCUSSION TITLE
What font is used in the demo picture?

### DISCUSSION THREAD

**Discussion by bluthej** [Textualize/rich#2895] [General]
Hi,

This is a bit silly but I am just wondering what's the font in [the demo picture](https://github.com/textualize/rich/raw/master/imgs/features.png)? 

I like it 😅

**Comment by willmcgugan** [comment]
Don't recall I'm afraid. I like to change my font from time to time.

**Reply by bluthej** [reply]
Too bad 😅
Thx anyway!

**Comment by bluthej** [comment]
Ok so I should have thought of that earlier but I ended up going to the [NerdFonts](https://www.nerdfonts.com/font-downloads) website and I'm pretty sure I found it!

In case other people are curious, I'm almost positive you used **FiraCode**.
